boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.dts

```diff
@@ -127,6 +127,12 @@
 		enable-gpios = <&gpio3 15 GPIO_ACTIVE_HIGH>;
 		regulator-boot-on;
 	};
+
+	isp: isp {
```

Comment:
even if not used, doesn't this have an address and belong to `soc/` node?

Reply:
no it is not have an address.

Reply:
so then, what is this node describing exactly? If you really need to manipulate some random GPIOs that turn out to have a predefined functionality (but not connected to any peripheral, like say I2C), why not use the GPIO API, just storing them as `*-gpios` in DT?

Reply:
@gmarull, any additional comments your end.